agent/consul/state/txn_test.go

```diff
@@ -577,6 +577,22 @@ func TestStateStore_Txn_KVS(t *testing.T) {
 				},
 			},
 		},
+		&structs.TxnOp{
+			KV: &structs.TxnKVOp{
+				Verb: api.KVGetNotExists,
+				DirEnt: structs.DirEntry{
+					Key: "foo/update",
+				},
+			},
+		},
+		&structs.TxnOp{
+			KV: &structs.TxnKVOp{
+				Verb: api.KVGetNotExists,
+				DirEnt: structs.DirEntry{
+					Key: "foo/not-exists",
```

Comment:
nit: do you think it would be clearer if we comment that we expect the Value to be nil here explicitly rather than just omitting it?